reinforcement_learning/rl_game_server_autopilot/ap-server/app.py

```diff
@@ -0,0 +1,228 @@
+from chalice import Chalice
+import json
+import numpy as np
+from time import gmtime,strftime
+import time
+from datetime import datetime
+from urllib.parse import urlparse, parse_qs
+import boto3
+import sys
+import os
+import requests
+
+#Global Varibale from model
+content_type = "application/json"
+accept = "Accept"
+min_servers=10
+max_servers=100
+new_alloc=0
+region='us-west-2'
+
+sagemaker_region = 'us-west-2'
+sagemaker_client = boto3.client('sagemaker-runtime',region_name=sagemaker_region)
+cloudwatch_region = 'us-west-2'
+cloudwatch_cli = boto3.client('cloudwatch',region_name=cloudwatch_region)
+
+app = Chalice(app_name='autopilot-server')
+
+#TBD Export to environment variables
+endpoint_name ="sagemaker-tensorflow-serving-2019-09-23-20-53-20-237"
+namespace = "autopilot"
+gs_inventory_url = "https://3vwufqstq5.execute-api.us-west-2.amazonaws.com/api/currsine24h"
```

Comment:
nit: we can move this to a separate config file instead of hardcoding here. That way all changes I would need to run against my account are in config file